Test-Source/Invoke-DockerLogin.Tests.ps1

```diff
@@ -18,11 +16,15 @@ Describe 'Login failure' {
             Mock -CommandName "Assert-ExitCodeOK" $assertExitCodeOkMocked -Verifiable -ModuleName $Global:ModuleName
         }
 
-        it 'should mask password from the logs when login fails after thrown exception' {
-            $loginCode = { Invoke-DockerLogin -Username "Mocked" -Password (ConvertTo-SecureString 'MockedPassword' –asplaintext –force) }
-            $loginCode | Should -Throw -ExceptionType ([System.Exception]) -PassThru
-            $result = GetMockValue -Key 'maskedCommand'
-            $result | Should -BeExactly 'Write-Output "*********" | docker login --username "Mocked" --password-stdin'
+        It 'should mask password from the logs when login fails after thrown exception' {
+            try {
+                Invoke-DockerLogin -Username "Mocked" -Password (ConvertTo-SecureString 'MockedPassword' –asplaintext –force)
+            } catch [System.Exception] {
+                $exception = $PSItem.Exception
```

Comment:
Curious. Can `$_` replace `$PSItem.Exception`? Perhaps `$PSItem.Exception` is more readable.

Reply:
Yes, they are interchangeable in this context. I prefer $PSItem to $_ as I always find myself spending a bit of energy remembering what $_ represents before I can read the code.